server/lib/nl/constants.py

```diff
@@ -263,6 +263,7 @@
                 "loss",
             ],
         },
+        "Overview": ["(tell me|more) about",],
```

Comment:
lets just stick to "tell me about" per guha's last comment?

Reply:
Done!